instrumentation/internal/internal-lambda/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/internal/lambda/InnerClassLambdaMetafactoryInstrumentation.java

```diff
@@ -89,7 +90,7 @@ private static class MetaFactoryClassVisitor extends ClassVisitor {
     private final String slashClassName;
 
     MetaFactoryClassVisitor(ClassVisitor cv, String slashClassName) {
-      super(Opcodes.ASM7, cv);
+      super(AsmApi.VERSION, cv);
```

Comment:
can you try rebasing, it's weird that github is showing these in the diff

Reply:
@laurit committed this change.